nipype/pipeline/plugins/sgegraph.py

```diff
@@ -136,7 +136,7 @@ def make_job_name(jobnumber, nodeslist):
                     if self._qsub_args.count('-o ') == 0:
                         stdoutFile = '-o {outFile}'.format(
                             outFile=batchscriptoutfile)
-                    full_line = '{jobNm}=$(qsub {outFileOption} {errFileOption} {extraQSubArgs} {dependantIndex} -N {jobNm} {batchscript} | awk \'{{print $3}}\')\n'.format(
+                    full_line = '{jobNm}=$(qsub {outFileOption} {errFileOption} {extraQSubArgs} {dependantIndex} -N {jobNm} bash {batchscript} | awk \'{{print $3}}\')\n'.format(
```

Comment:
@ostanley @satra I think it would be better to ensure that that bash script has execute permisssions.